frontend/apps/marketing/src/components/contentful/collections/peopleCollection/PeopleCollection.tsx

```diff
@@ -0,0 +1,165 @@
+import OpenInNew from '@mui/icons-material/OpenInNew';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import {EntryFields} from 'contentful';
+import {useMemo, useId} from 'react';
+
+import {getAbsoluteImageUrl} from '@/selectors/contentful/getImage';
+import {LinkEntry} from '@/types/contentful/entries/Link';
+import {Entry} from '@/types/contentful/Entry';
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+type ItemFields = {
+  name: EntryFields.Text;
+  title: EntryFields.Text;
+  image?: ExperienceAsset;
+  bio?: EntryFields.Text;
+  personalLink?: LinkEntry;
+};
+
+type ItemEntry = Entry<ItemFields>;
+
+export type PeopleCollectionProps = {
+  /** Collection content w/ fields from Contentful */
+  people: ItemEntry[];
+  /** Show or hide images */
+  imageVisibility?: 'show' | 'hide';
+};
+
+const styles = {
+  container: {
+    alignItems: 'flex-start',
+  },
+  gridItem: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    textAlign: 'center',
+  },
+  image: {
+    marginBottom: '1.25rem',
+    width: 128,
+    height: 128,
+    borderRadius: '50%',
+  },
+  overline: {
+    color: '#4C5661',
+    marginTop: 1,
+    marginBottom: 1.5,
+  },
+  bio: {
+    textAlign: 'center',
+  },
+  personalLink: {
+    display: 'inline-flex',
+    alignItems: 'center',
+    gap: 0.5,
+    marginTop: 1,
+    'html[dir="rtl"] & svg': {
+      transform: 'scaleX(-1)',
+    },
+  },
+};
+
+// This is coming directly from Contentful so we can use the Images API to
+// format this similarly to the other images which is better for performance.
+const PLACEHOLDER_IMAGE_URL =
+  'https://contentful-images.code.org/90t6bu6vlf76/12hYaT6lKHqt7yO3knvqHF/8cc3dc67771cd6baaecdff1b155b7d31/person-placeholder.png?fm=avif&fit=fill&w=128&h=128&r=max';
```

Comment:
Open to suggestions on how to handle this. I tried not using the Images API for the placeholder and entry images but the images were slower to load. I also tried applying a Skeleton but it requires CSR to get the loading state.

Reply:
I would take the image, put it in the `peopleCollection` directory. Then import the image as suggested above/below to keep the image local